lambda/app/src/controllers/user.ts

```diff
@@ -286,14 +286,10 @@ export const deleteStaleUsers = async (user: any) => {
             rqst.archived = true;
             if (rqst.status !== 'draft') {
               rqst.status = 'submitted';
-              const ghResult = await dispatchRequestWorkflow(rqst);
-
-              if (ghResult.status !== 204) {
-                throw Error('failed to create a workflow dispatch event');
-              }
 
               await disableIntegration(rqst.get({ plain: true, clone: true }));
-              await closeOpenPullRequests(rqst.id);
+              await dispatchRequestWorkflow(rqst);
+              //await closeOpenPullRequests(rqst.id);
```

Comment:
We can probably remove this commented line

Reply:
Removed